index.html

```diff
@@ -33,17 +33,19 @@ <h1><span class="H1 capital">J</span>enny <span class="H1 capital">T</span>schau
                 <h3 id="About-me" class="header-box-1">About me</h3>
                     <p class="text-box-1">Lorem ipsum, dolor sit amet consectetur adipisicing elit. Necessitatibus sunt sit saepe accusantium asperiores dolores voluptatem ullam impedit quod consequuntur quaerat, pariatur minus! A saepe eligendi modi ex, quam nisi. Lorem ipsum dolor sit amet consectetur adipisicing elit. Ea numquam consectetur quaerat magni repellendus quod laborum iure nesciunt delectus suscipit repudiandae eum laudantium cum sapiente, alias facilis obcaecati provident dolore.
                     <br><br> 
-                    Lorem ipsum dolor sit amet consectetur adipisicing elit. Labore delectus, odit modi minus saepe voluptas incidunt dignissimos, explicabo velit animi accusantium! Obcaecati quod vero beatae, quas hic reiciendis architecto placeat! Lorem ipsum dolor sit amet consectetur adipisicing elit. Deleniti voluptatum optio eligendi tempore voluptatibus omnis ipsum, quisquam magnam minima? Vitae quod beatae eos iure soluta veniam nemo sapiente non asperiores.</p>
+                    Lorem ipsum dolor sit amet consectetur adipisicing elit. Labore delectus, odit modi minus saepe voluptas incidunt dignissimos, explicabo velit animi accusantium! Obcaecati quod vero beatae, quas hic reiciendis architecto placeat! Lorem ipsum dolor sit amet consectetur adipisicing elit.</p>
+                    <br>
+                    <h2 id="check-out">-> Check out my <a href="./pdf/Profile.pdf"id="CV">CV</a></h2> 
```

Comment:
At the moment, this takes me to the 404 page, I think if you update the path though to go through the assets folder it should fix it :-)